### PR TITLE
People flyout 2

### PR DESCRIPTION
**Pull request by ralphlevan**
Flyout now displays the table of surveys assigned to that person. Still TODO: clicking on the survey shows their responses

### CODE REVIEW

**Comment by ralphlevan** [comment]
Crap! Sorry! I keep missing the part in the PR form where it says I'm merging into main instead of development

File: app/javascript/people/people_sidebar.vue [hunk 1]
```diff
@@ -15,7 +15,7 @@
           <people-admin-tab :model="model"></people-admin-tab>
         </b-tab>
         <b-tab title="Surveys">
-          <people-surveys-tab :model="model"></people-surveys-tab>
+          <people-surveys-tab :model="model" :selected-person="selected"></people-surveys-tab>
```
Comment (Gailbear) [inline app/javascript/people/people_sidebar.vue:18]: you don't need the selected person here - with the model mixin, it comes with this.selected, which will be the same thing.

File: app/javascript/people/people_surveys_tab.vue [hunk 2]
```diff
@@ -8,22 +8,31 @@
...
 import SurveyTable from '../surveys/survey_table';
+import personSessionMixin from "@/auth/person_session.mixin";
 
 export default {
   name: 'PeopleSurveysTab',
   components: {
     SurveyTable
   },
+  props: {
+    selectedPerson: Object
```
Comment (Gailbear) [inline app/javascript/people/people_surveys_tab.vue:27]: don't need - see above

File: app/javascript/people/people_surveys_tab.vue [hunk 3]
```diff
@@ -8,22 +8,31 @@
...
+  },
+  computed: {
+    queryString() {
+      return '{"queries":[["assigned_people.person_id", "=", "'+this.selectedPerson.id+'"]]}';
+    }
+  },
   mixins: [
-    modelMixin
+    modelMixin, personSessionMixin
```
Comment (Gailbear) [inline app/javascript/people/people_surveys_tab.vue:35]: where are you using personSessionMixin?

**Review by Gailbear — COMMENTED** [review]
great start :)

**Comment by ralphlevan** [comment]
Good suggestions. I didn't know that value was there.

The personSessionMixin is a leftover from (probably) the user table that I
cloned to make that.

File: app/javascript/people/people_surveys_tab.vue (hunk 3 above)
Comment (ralphlevan) [reply app/javascript/people/people_surveys_tab.vue:35]: Leftover from the cut-and-paste. There was a commented out section in the template that used that data. The mixin and the commented section will go away

**Pull request merged**